src/chapters/arrays/array-methods.rst

```diff
@@ -0,0 +1,148 @@
+.. _array-methods:
+
+Array Methods
+==============
+
+.. index::
+   single: array; methods
+
+As with strings, JavaScript provides us with useful **methods** for arrays.
+Unlike strings, arrays are *mutable* and can be changed.
+
+Some methods alter an array, while others *return* values.
+
```

Comment:
Rethinking this intro so it flows better. What about:

"As with strings, JavaScript provides us with useful methods for arrays. Unlike strings, arrays are mutable and can be changed without creating a new array. This means that unlike string methods, many array methods do not return a new array, but rather alter the array. Some array methods alter an array, others return values, and a few do return a new array. 

Below are some common array methods."

Reply:
Hmmm.  We went from about 18 words to... more, and I think some clarity was lost.

Re-reading the paragraph, I'm not sure what we gain.  Are we trying to evoke a stronger contrast with strings?  Are we trying to distinguish between *returning* a new array vs. *mutating* an existing array?

My first (and strongest) preference is to keep the original text in place.  It's shorter and does the job.

If we are trying to expound upon the potential, varied results stemming from the application of the different array methods, how about:
"As with strings, JavaScript provides us with useful methods for arrays.  These methods will either *alter* an existing array, *return* information about the array, or *create and return* a new array."

If we need to bring in a string contrast, then how about something like, "While string methods never change the string itself..."

Reply:
Sounds fine.

Reply:
OK.  I think Blake started the PR, so he will need to close and merge.